ci/environment_linux.yml

```diff
@@ -0,0 +1,20 @@
+name: lf
```

Comment:
Let's not add another environment file. Instead let's reuse the existing `ci/environment.yml` file similar to other jobs.